src/modules/external_services/exceptions.py

```diff
@@ -0,0 +1,7 @@
+class ExternalServiceFailure(Exception):
+    def __init__(self):
+        pass
+
+class ExternalProviderFailure(Exception):
+    def __init__(self, exc=None):
+        self.wrapped_exc = exc
```

Comment:
The difference between these exceptions is not clear (I understood that after reading the using code).
Please add docstring that will clarify the usage and difference